src/rpc/handlers/Ledger.hpp

```diff
@@ -102,7 +103,9 @@ class LedgerHandler {
     {
         static auto const rpcSpec = RpcSpec{
             {JS(full), validation::Type<bool>{}, validation::NotSupported{true}},
```

Comment:
When full is true, Clio handler would not be called. We can remove the NotSupported part. Warning is only given when full is false, right?

Reply:
I think we should leave NotSupported, in case logic in forwarded part would change. 

Warning is given on any full value if Clio is processing the request. But because we are forwarding on `full: true`, yeah, only when full is false the warning will be given.